charts/tikv-importer/templates/tikv-importer-configmap.yaml

```diff
@@ -3,9 +3,9 @@ kind: ConfigMap
 metadata:
   name: {{ .Values.clusterName }}-importer-{{ template "importer-configmap.data-digest" . }}
   labels:
-    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/name: {{ .Values.clusterName }}-importer
```

Comment:
I think we do not need to change this, just change the instance label is OK.

Reply:
reverted